ofl/udgothic/METADATA.pb

```diff
@@ -0,0 +1,29 @@
+name: "UD Gothic"
+designer: "Type Bank Co., Morisawa Inc."
+license: "OFL"
+category: "SANS_SERIF"
+date_added: "2022-03-15"
+fonts {
+  name: "UD Gothic"
+  style: "normal"
+  weight: 400
+  filename: "BIZUDGothic-Regular.ttf"
+  post_script_name: "BIZUDGothic-Regular"
+  full_name: "BIZ UDGothic"
+  copyright: "Copyright 2022 The BIZ UDGothic Project Authors (https://github.com/googlefonts/morisawa-biz-ud-gothic)"
+}
+fonts {
+  name: "UD Gothic"
+  style: "normal"
+  weight: 700
+  filename: "BIZUDGothic-Bold.ttf"
+  post_script_name: "BIZUDGothic-Bold"
+  full_name: "BIZ UDGothic Bold"
+  copyright: "Copyright 2022 The BIZ UDGothic Project Authors (https://github.com/googlefonts/morisawa-biz-ud-gothic)"
+}
+subsets: "cyrillic"
+subsets: "greek-ext"
+subsets: "japanese"
+subsets: "latin"
+subsets: "latin-ext"
+subsets: "menu"
```

Comment:
A lot of missing information in METADATA.pb, did you upgrade gftools before making the PR?

Reply:
I thought I had, but apparently not! I've rebuilt these. (and fixed a gftools bug along the way :) )